src/praisonai/tests/unit/test_external_agents_backcompat.py

```
@@ -0,0 +1,124 @@
+    from praisonai.cli.features.external_agents import ExternalAgentsHandler
+    
+    handler = ExternalAgentsHandler(verbose=False)
+    
+    # Mock the integration to avoid actual subprocess calls
+    with patch.object(handler, 'get_integration') as mock_get_integration:
+        mock_integration = MagicMock()
+        mock_integration.is_available = True
+        mock_integration.cli_command = 'claude'
+        mock_get_integration.return_value = mock_integration
+        
+        integration = handler.get_integration('claude', workspace='/tmp')
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)